### ISSUE TITLE
TitleBar: Unable to select any color from the Background/foreground dropdown

### ISSUE THREAD

**Issue by gregwoo-microsoft** [microsoft/WinUI-Gallery#1071]
### Which version of the app?

WinUI 3 Gallery

### Description

**Repro-Steps:**
Launch the WinUI app
Navigate to search box using tab key
Type 'TitleBar' in search box and hit enter.
Now navigate to TitleBar card and activate it.
Navigate to Background/foreground color and expand it.
Now try to select any color from the button and observe.
Actual Result:
Unable to select any color from the Background/foreground dropdown.

**Expected Result:**
User should be able to select any color from the Background/foreground dropdown.

**User Impact:**
Keyboard user will not be able to select and consume the color from the dropdown.

### Windows version

Windows 11 22H2 (22621)
